### PR TITLE
Dialog Implementation

### PR DESCRIPTION
**Pull request by 8leak**
* Added dialog system UI
* Added conversation class for managing interface with Clyde interpreter
* Added Clyde dialog Godot addon

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
+# Clyde Interpreter Usage
```
Comment (verillious) [inline USAGE.md]: Should be in the addons folder

**Review by verillious — CHANGES_REQUESTED** [review]
Move the USAGE.md file into the addons folder

File: USAGE.md (hunk 1 above)
Comment (8leak) [reply USAGE.md]: thanks, fixed

**Review by verillious — APPROVED** [review]
LGTM

**Pull request merged**